regions/io/fits/tests/data/region.fits

```diff
@@ -0,0 +1 @@
�b�@����e�BOX             @�]ֻ���@�x��!�@<e*b�D @�     @�!     ELLIPSE         @r      @�      @uz��� @uP     @u�     ROTBOX          @L      @S�     @P@      @uP     @u�     ANNULUS         @L      @S�              @uP     @u�     POINT                                    @uP     @u�                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
```

Comment:
When I read the WCS object from the header of hdul[1] I get a warning

`FITSFIxedWarning: the RADECSYS keyword is deprecated, use RADESYSa`

Reply:
This is the official sample which seems to be old.

Reply:
Also, I tried looking into the header and it seems different typical header (as shown in the doc examples).  The `WCS` object that I get from it seems to be bad. Can you look into it and say whether astropy does not support  it or it's too old and deprecated?

Reply:
any difference between `TCTYP1` vs `CTYPE1` ?

Reply:
I'm finding this problem with the WCS object using astropy 3.0.4. Is this the FITS file available from https://fits.gsfc.nasa.gov/registry/region.html ?

Reply:
The `T` prefix implies that it's associated with a binary table HDU, while the lack of a preceding `T` implies that it's associated with an image HDU

Reply:
@migueldvb yes the file is from the site. To me also it seems to be a problem with WCS. It is not able to read the T prefix keyword, but according to @keflavich it should this??

Reply:
It seems that the regions.fits file does not have celestial component needed to find the position on the sky sphere.
```
In [26]: wcs
Out[26]: 
WCS Keywords

Number of WCS axes: 2
CTYPE : ''  ''  
CRVAL : 0.0  0.0  
CRPIX : 0.0  0.0  
PC1_1 PC1_2  : 1.0  0.0  
PC2_1 PC2_2  : 0.0  1.0  
CDELT : 1.0  1.0  
NAXIS : 58  7

In [28]:  wcs = wcs.sub([WCSSUB_LONGITUDE, WCSSUB_LATITUDE])
    ...: 

In [29]: wcs
Out[29]: 
WCS Keywords

Number of WCS axes: 0
CTYPE : 
CRVAL : 
CRPIX : 
CDELT : 
NAXIS : 58  7
```

Perhaps the `read_fits_region` could return the pixel regions, or catch the exception when the transformation to sky coordinates fails and give an error message.

Reply:
Interesting, the first extension HDU (hdul[1]) is a binary table so it should require the `T` prefix, but great that it works!

Reply:
Do you think we should raise this issue in the astropy core?

Reply:
I just saw that `astropy.wcs.WCS` has a `keysel` optional argument that may be required for binary tables, but it is not working for me. Maybe report this problem in the wcs channel on slack?

Reply:
good idea!

Reply:
```
wcs = WCS(f[1].header, keysel=['image','binary','pixel'])
```
This works! Thanks for finding this out

Reply:
great! I didn't try that combination.